### PR TITLE
Add Verification Instructions to Slack Message

### PR DESCRIPTION
**Pull request by nlordell**
Just a small suggestion - I found it could be nice to also have a link to the verification instructions so you can open everything you need directly from the Slack thread with the payout transaction.

Feel free to close the PR if you don't agree with the change.

### Test Plan

Not sure how to test this change without actually posting something on Slack.

### CODE REVIEW

**Comment by nlordell** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by bh2smith — APPROVED** [review]
This is absolutely incredible! Thanks very much for adding this. Just a small lint error that you can fix by running `make check` on the repo.

**Comment by nlordell** [comment]
> Trailing whitespace

😱 - I'm usually so picky with this...

File: src/fetch/dune.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 log = set_log(__name__)
 
+
```
Comment (nlordell) [inline src/fetch/dune.py:24]: `make fmt` did this.

File: tests/e2e/test_prices.py [hunk 2]
```diff
@@ -40,7 +40,6 @@ def test_token_in_usd(self):
         )
 
     def test_eth_in_token(self):
-
```
Comment (nlordell) [inline tests/e2e/test_prices.py:43]: And this

File: tests/e2e/test_prices.py [hunk 3]
```diff
@@ -53,7 +52,6 @@ def test_eth_in_token(self):
         )
 
     def test_token_in_eth(self):
-
```
Comment (nlordell) [inline tests/e2e/test_prices.py:56]: and this.

File: src/fetch/dune.py (hunk 1 above)
Comment (bh2smith) [reply src/fetch/dune.py:24]: Seems fine, Maybe a different python version...?

File: tests/e2e/test_prices.py (hunk 2 above)
Comment (bh2smith) [reply tests/e2e/test_prices.py:43]: cool with me.

**Pull request merged**